core/store/models/job_spec_error.go

```diff
@@ -0,0 +1,25 @@
+package models
+
+import (
+	"time"
+)
+
+// JobSpecError represents an asynchronous error caused by a JobSpec
+type JobSpecError struct {
+	ID          *ID       `json:"id"`
+	JobSpecID   *ID       `json:"-"`
+	Description string    `json:"description"`
+	Occurances  uint      `json:"occurances"`
```

Comment:
In the database its an integer type, so I suggest usint `int32` here.

Also, spelling.